src/avram/factory.cr

```diff
@@ -31,10 +37,6 @@ abstract class Avram::Box
     end
   end
 
-  def self.save
-    {% raise "'Box.save' has been renamed to 'Box.create' to match 'SaveOperation.create'" %}
-  end
```

Comment:
This has been removed for quite a few versions now, so just getting rid of it instead of fixing the error message.